File: master/plotter.py
import csv
import numpy as np
import matplotlib.pyplot as plt

#%%
file = open(r'F:\Guth data\17092020 12.csv')
data_array = csv.reader(file)
zipped_data = list(data_array)

#%%
# Unzip list
data = [[i for i, j, k, l in zipped_data],
            [j for i, j, k, l in zipped_data],
            [k for i, j, k, l in zipped_data],
            [l for i, j, k, l in zipped_data]]

time = np.array(data[0])
force = np.array(data[1])
length = np.array(data[2])
signal = np.array(data[3])
#%%
fig = plt.figure()
ax = fig.add_axes([0, 900, 0, 10])
ax.plot(time, force)
ax.set_xticklabels('')
ax.set_yticklabels('')
plt.show()

File: master/plotter.jl
using CSV
using GR, Interact
data = CSV.read("F:\\Guth data\\17092020 12.csv")
timedata = data[:, 1]
force = data[:, 2]
length = data[:, 3]
signal = data[:, 4]

t = plot(timedata, force)
gui()
plot(timedata, length)
plot(timedata, signal)
